README.md

```diff
@@ -43,45 +43,44 @@ behavior, but if you have a different use case,
 
 ```
 $ cloudflare-dyndns --help
-Usage: cloudflare-dyndns [OPTIONS] [DOMAINS]...
+Usage: python -m cloudflare_dyndns.cli [OPTIONS] [DOMAINS]...
```

Comment:
@kissgyorgy, you might want to look into https://github.com/zimbatm/mdsh to help contributors remember to update this.

Reply:
Thanks, I will look into the tool.